okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/DefaultHostMetrics.java

```diff
@@ -122,9 +134,11 @@ void record(int statusCode, long micros) {
                 other.update(micros, MICROS);
                 break;
         }
+        lastUpdate = Instant.now(clock);
```

Comment:
nit: assuming we're updating this much more frequently than we're querying it, maybe remember epoch millis and create the `Instant` in `lastUpdate()`?

Reply:
done